contracts/Bridge.sol

```diff
@@ -159,6 +159,18 @@ contract Bridge is Pausable, Context, EIP712 {
         handler.setBurnable(tokenAddress);
     }
 
+      /**
+        @notice Sets source and destination number of decimals for handler contract and provided token,
+        which is mapped in {_decimals} in ERCHandlerHelpers.
+        @notice Only callable by address that has the right to call the specific function.
+        @param handlerAddress Address of handler resource will be set for.
+        @param tokenAddress Address of contract to be called when a deposit is made and a deposited is executed.
+     */
+    function adminSetDecimals(address handlerAddress, address tokenAddress, uint8 srcDecimals, uint8 destDecimals) external onlyAllowed {
+        IERCHandler handler = IERCHandler(handlerAddress);
+        handler.setDecimals(tokenAddress, srcDecimals, destDecimals);
```

Comment:
Source/Destination notation sounds vague, because it implies the movement from source to destination, while we are interested in tokens coming here from somewhere. So maybe we call it localDecimals and externalDecimals? Which raises a question, could there be a situation where particular token is bridged between 3 chains, and each chain has different decimals value? if yes then current solution wouldn't work, and instead we could only look at local decimals (which is already available in the token contract), while assuming that external decimals are always 18. Then whenever we emit events on the amounts, we convert them to 18 decimals?